### PR TITLE
Feature/landingpage design improvements

### PR DESCRIPTION
**Pull request by Ben-Zahler**
This adds some design improvements on the landing page components used in AEM.
The changes were created in a live session with Jonas Güther

## Type of change

- [x] New feature (non-breaking change which adds functionality)
 
 # How Has This Been Tested?
 
tested on local instance
 
 # Checklist:
 
 - [x] My code follows the style guidelines of this project
 - [x] I have performed a self-review of my own code
 - [ ] I have commented my code, particularly in hard-to-understand areas
 - [ ] I have made corresponding changes to the documentation
 - [x] My changes generate no new warnings
 - [ ] I have added tests that prove my fix is effective or that my feature works
 - [ ] New and existing unit tests pass locally with my changes
 - [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: src/components/o-commercial-hero-cover/index.scss [hunk 1]
```diff
@@ -61,8 +61,8 @@ $margin-30: 30px;
     }
   }
 
-  @include respond-up(xl) {
-    height: $hero-xl-height;
+  @include respond-up(xxl) {
```
Comment (AndyOGo) [inline src/components/o-commercial-hero-cover/index.scss]: We don't have an `xxl` breakpoint

File: src/styles/grid/_grid-settings.scss [hunk 2]
```diff
@@ -21,7 +21,9 @@ $grid-breakpoints:
     // Large screen / desktop
     lg: 992px,
     // Extra large screen / wide desktop
-    xl: 1200px
+    xl: 1200px,
+    // Extra large screen / wide desktop
```
Comment (AndyOGo) [inline src/styles/grid/_grid-settings.scss]: I would rename that comment

File: src/styles/grid/_grid-settings.scss [hunk 3]
```diff
@@ -31,5 +33,6 @@ $container-max-widths:
     sm: 540px,
     md: 720px,
     lg: 960px,
-    xl: 1140px
+    xl: 1140px,
+    xxl: 1360px
```
[the remark this answers is not in the document]
Comment (Ben-Zahler) [reply src/styles/grid/_grid-settings.scss]: no, I assumed that if I add a new breakpoint, I also should change the container max-width, but now that seems to not make that much sense... change is removed

File: src/styles/grid/_grid-settings.scss (hunk 2 above)
Comment (Ben-Zahler) [reply src/styles/grid/_grid-settings.scss]: better now?

File: src/components/o-commercial-hero-cover/index.scss [hunk 4]
```diff
@@ -9,8 +9,8 @@
 $picture-z-index: -1;
 $hero-mobile-height: 256px;
```
[the remark this answers is not in the document]
Comment (Ben-Zahler) [reply src/components/o-commercial-hero-cover/index.scss]: done

**Pull request merged**